### PR TITLE
I4 i6 setup improvements

### LINKED ISSUE #4: Separate docker services while IPE main process and test
Will have 2 docker service  for running main process and test

docker compose up job
docker compose up test
docker compose up - run both test and job
------------------------------------------------------------------------------------------
Code Review comments
I think I was just saying rather than a switch in the start script to have two script files like
run_tests.sh
start.sh
This is kind of the pattern that canvasapi is using https://github.com/ucfopen/canvasapi/tree/develop/scripts
Code review comments by @jonespm 
 https://github.com/tl-its-umich-edu/IPE-competencies-updater/pull/3/files#r728634011

### LINKED ISSUE #6: Improve the google service account path for running the script
Remove the IS_DOCKER_RUN variable and instead have the absolute path to the google service account file. this way support running in docker or command line( this is convenient)

For Docker run set this
`GSERVICE_ACCOUNT_PATH=/secrets/ipe/ipe_gserviceaccount.json`
For Running via command line
`GSERVICE_ACCOUNT_PATH=/user/home_directory/secrets/ipe/ipe_gserviceaccount.json
`

----------------------------------------------------------------------
Could probably create this as an issue and could improve it in the future. I have some test scripts that just detected the location of the file and used that as a reference point.

https://github.com/tl-its-umich-edu/api-utils-python/blob/5212d2a30664acb78b5b74c8a25296bced87f2a2/tests/test.py#L10

Comment by @jonespm

### PR DESCRIPTION
**Pull request by pushyamig**
Fixes #4 Fixes #6 

1. Try updating your .env with absolute path to you docker container which is `/secrets/ipe/ipe_gserviceaccount.json`. If running on command line, use the absolute path to the directory
2. I have updated the sheet id in .env_sample. please use that for running pytest otherwise test results might vary.
3. `docker compose up job` for running Main process
4. `docker compose up test` running pytest
5. simple `docker compose up` will run both, but might confuse you

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -1,5 +1,6 @@
+-e git+https://github.com/tl-its-umich-edu/api-utils-python@v2.0#egg=umich-api
 pandas==1.3.3
-python-dotenv==0.19.0
+python-dotenv==0.14.0
```
Comment (pushyamig) [inline requirements.txt:3]: api-utils-python library is still on 0.14 and if I have updated to 0.19 in my `requirements.txt` pip install fails. So Downgrading for now. Either I have to update the `api-utils-python` library or figure out an approach. Thoughts??

File: config/env_sample.txt [hunk 2]
```diff
@@ -1,8 +1,5 @@
-IPE_SHEET_ID=1fvYhtxlu3pa8vNBjOVzp1hwNBsv823mTD3uP6l5_WuI
-IPE_SHEET_NAME=Offering
-
-GSERVICE_ACCOUNT_PATH=secrets/ipe/ipe_gserviceaccount.json
+IPE_SHEET_ID=1UrBaZpR5USS-DbbsdNNzjfEFdLQSnqg5RKUUmHkXHGg
```
Comment (zqian) [inline config/env_sample.txt]: use a place holder for the GSheet ID

File: requirements.txt (hunk 1 above)
Comment (zqian) [reply requirements.txt:3]: +1 for upgrade python-dotenv in api-utils-python

**Review by zqian — COMMENTED** [review]
I have two suggestions. Otherwise, I have run job and test successfully.

File: requirements.txt (hunk 1 above)
Comment (pushyamig) [reply requirements.txt:3]: OK, I will update it some point before go Live. I have created issue as of now https://github.com/tl-its-umich-edu/api-utils-python/issues/39

File: config/env_sample.txt (hunk 2 above)
Comment (pushyamig) [reply config/env_sample.txt]: I think I resolved this from that I understood you wanted and added some comments around this

**Review by zqian — APPROVED** [review]

**Pull request merged**